src/WatchDogs.Persistence.EntityFramework/DataInserter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WatchDogs.Contracts;
+
+namespace WatchDogs.Persistence.EntityFramework;
+public class DataInserter : IDataInserter
+{
+    private readonly ApplicationDbContext _context;
+
+    public DataInserter(ApplicationDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task InsertTradeDatatoDbAsync(IEnumerable<Trade> data)
+    {
+        foreach (var trade in data)
+        {
+            await _context.Trades.AddAsync(trade);
+        }
+
+        await _context.SaveChangesAsync();
+    }
+    public async Task InsertTradeDatatoDbAsyncccc(IEnumerable<Trade> data)
```

Comment:
The implementation is good but name is retarded 😆 Please rename the method and remove the unused one

Reply:
Done.